LUIS/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+// This is a template webpack config file with minimal configuration.
+// Users are free to create their own webpack configuration files in their application.
+const webpack = require('webpack');
+const path = require('path');
+module.exports = {
+  entry: './lib/luisAuthoring.ts',
+  devtool: 'source-map',
+  output: {
+    filename: 'luisAuthoringBundle.js',
+    path: __dirname,
+    libraryTarget: 'var',
+    library: 'luisAuthoring'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+        exclude: /(node_modules|test)/
+      }
+    ]
+  },
+  // "ms-rest-js" and "ms-rest-azure-js" are dependencies of this library.
+  // Customer is expected to import/include this library in browser javascript
+  // (probably using the script tag in their html file).
+  externals: {
+    "ms-rest-js": "msRest",
+    "ms-rest-azure-js": "msRestAzure"
+  },
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"]
+  },
+  node: {
```

Comment:
If this is necessary, it means some node libs are being included in the dependency tree. We should be sure this won't introduce side-effects